trollmoves/client.py

```diff
@@ -234,7 +235,11 @@ def resend_if_local(msg, publisher):
 
 def create_push_req_message(msg, destination, login):
     fake_req = Message(msg.subject, 'push', data=msg.data.copy())
-    duri = urlparse(destination)
+    try:
+        _destination = compose(destination, msg.data)
+    except Exception:
```

Comment:
I think this exception can only be:
 - `KeyError` when format identifier is missing from `msg.data`
 - `ValueError` when type of format identifier doesn't match the type in `msg.data`
 - `AttributeError` when `msg` is None (I'm not sure this can even happen)
Anyway, the exlpicit errors should be used.

Reply:
Yep. Agree